include/openssl/crmf.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This is a new file. Is the claim of copyright back to 1995 justified? Similarly for the Nokia and Siemens copyright claims below.

Reply:
OpenSSL copyright might be a tricky question: the CMP/CRMF code builds up and loans a lot from existing OpenSSL code, which generally states copyright going back to 1995. It might be of course OK to adjust that to the earliest version of the file, which will usually be 2007?

The dates for the Nokia and Siemens copyright claims are verified; development was started in 2007.

Reply:
I'm fine with whatever the earliest date is.